lib/bloomr/credit.rb

```diff
@@ -4,8 +4,6 @@ module Bloomr
-        variables: { ORDER_ID: "#{order_id}", order_id: "#{order_id}" }
```

Comment:
This is the GraphQL query to pull the full credit report.
Is this not needed for #get_credit_data() to return something meaningful?

Reply:
@jiahou here, the all being made is the `/full-report` endpoint (see https://github.com/bloomcredit/bloomRuby/pull/6/commits/6c907da206cbbd79c6e73ff9fb764f666f2ebc0c#diff-09a1ae51fc9c016ad0e95bbe87c2d4b6e63d3e257c5bbdfbe528af241aa567d3L17) 

This endpoint doesn't require any body parameter (and in fact returns a 400 if any body payload is sent).

See further documentation https://developers.bloomcredit.io/reference/data-accessget-order-full-report